core/src/jvmMain/kotlin/com/ustadmobile/core/util/ShrinkUtils.kt

```diff
@@ -0,0 +1,98 @@
+package com.ustadmobile.core.util
+
+import com.github.aakira.napier.Napier
+import com.ustadmobile.core.catalog.contenttype.VideoTypePlugin
+import java.io.BufferedReader
+import java.io.File
+import java.io.IOException
+import java.io.InputStreamReader
+
+object ShrinkUtils {
+
+    fun findInPath(commandName: String, systemPropName: String? = null): String? {
```

Comment:
This wouldn't work on Windows (as windows commands would have .exe or .bat). I would suggest creating the following extension function for file:

```
/**
 * Checks to see if there is a valid command here. On Linux this is the same as file.exists(). On windows, this will look for the file itself as well as variants with .exe and .bat
 */
fun File.commandExists() {
    return if(linux) {
       exists()
     } else if(windows) }
        File(parent "$name.exe").exists() || File(parent, "$name.bat").exists()
     }
}
```